### ISSUE TITLE
This site can’t be reached

### ISSUE THREAD

**Issue by Anonymous-so** [karpathy/arxiv-sanity-preserver#150]
Does the server die?

**Comment by Ahacad** [comment]
Same problem:
```
This page isn’t working
www.arxiv-sanity.com didn’t send any data.
ERR_EMPTY_RESPONSE
```

**Comment by karpathy** [comment]
Yeah so now it's dying because I'm running out of RAM on the machine. I can't simultaneously host and re-compute SVMs for all 121K+ papers. Whole thing needs a refactor, sorry.

**Comment by dbalabka** [comment]
@karpathy I can provide some help with this. I would like to dig into this project because found it very useful.

**Comment by ryx2** [comment]
@karpathy I also would love to help with this project; use it all the time, have experience doing document recommendation. Is the current master the version being used to host the site?
